src/game/behaviors/falling_rising_platform.inc.c

```diff
@@ -3,9 +3,6 @@
 void bhv_squishable_platform_loop(void) {
     o->header.gfx.scale[1] = (sins(o->oBitfsPlatformTimer) + 1.0f) * 0.3f + 0.4f;
     o->oBitfsPlatformTimer += 0x80;
-#ifdef AUTO_COLLISION_DISTANCE
-    o->oFlags &= ~OBJ_FLAG_DONT_CALC_COLL_DIST;
-#endif
```

Comment:
what was this for?

Reply:
Previously the platform recalculated collision distance every frame. This just reverts it to vanilla behavior since it has OBJ_FLAG_DONT_CALC_COL_DISTANCE now.

Reply:
Previously, the scaling/squishing platform recalculated its collision distance. This reverts the collision distance to vanilla since it has OBJ_FLAG_DONT_CALC_COLL_DISTANCE now.